src/main/java/gov/nasa/pds/harvest/cfg/model/Configuration.java

```diff
@@ -14,6 +14,7 @@
     
     public List<BundleCfg> bundles;
     public List<String> dirs;
+    public List<String> manifests;
```

Comment:
Was there specific reason to directly use public variables here? It is generally not recommended to have public variables in Java classes due the to reasons mentioned in the following Sonar rule.

**Class variable fields should not have public accessibility**
https://rules.sonarsource.com/java/RSPEC-1104

It is recommended to use private attributes and public accessor methods (set and get).